<?xml version="1.0"?> <launch>
    <arg name="rgb" default="/rgb/image_raw" />
    <arg name="video_width" default="1280" />
    <arg name="video_height" default="720" />

    <arg name="gpuid" default="0" />

    <!--
    <arg name="model" default="mobilenet_thin" />
    -->
    <arg name="model" default="cmu" />
    <arg name="resolution" default="432x368" />

    <node name="pose_estimator" pkg="tfpose_ros" type="broadcaster_ros.py" output="screen" required="true">
        <env name="CUDA_VISIBLE_DEVICES" value="$(arg gpuid)" />
        <param name="camera" value="$(arg rgb)" />
        <param name="model" value="$(arg model)" />
        <param name="resolution" value="$(arg resolution)" />
    </node>

    <node name="pose_estimator_vis" pkg="tfpose_ros" type="visualization.py" output="screen" required="true">
        <param name="camera" value="$(arg rgb)" />
        <param name="pose" value="/pose_estimator/pose" />
    </node>
</launch>
